oshdb-filter/src/main/java/org/heigit/ohsome/oshdb/filter/IdRange.java

```diff
@@ -34,4 +35,8 @@ public String toString() {
         + ".."
         + (toId == Long.MAX_VALUE ? "" : toId);
   }
+
+  public LongStream getIds() {
```

Comment:
maybe add a short javadoc? e.g. 
```suggestion
  /** Converts the id range to a stream of consecutive ids. */
  public LongStream getIds() {
```